integration/default_test.go

```diff
@@ -69,8 +69,9 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 					procfileBuildpack,
 				).
 				WithEnv(map[string]string{
-					"BP_LOG_LEVEL":  "DEBUG",
-					"BP_PHP_SERVER": "nginx",
+					"BP_LOG_LEVEL":           "DEBUG",
+					"BP_PHP_SERVER":          "nginx",
+					"BP_NGINX_CONF_LOCATION": "/layers/paketo-buildpacks_php-nginx/php-nginx-config/nginx.conf",
```

Comment:
@TisVictress I think it would be strange for a user to have to know where the Nginx conf file goes on the filesystem, since it's the job of the buildpack to do that, rather than something configurable by the user. Instead of having this as a user-passed environment variable, we should have this set in the buildpack I think.

Reply:
Per the [original thread](https://paketobuildpacks.slack.com/archives/CUD6TBJAE/p1647977234891499?thread_ts=1647976317.655719&cid=CUD6TBJAE) on this, the environment variable is for the buildpack to tell the launcher where the nginx.conf file is. Then, the launcher process runs the `configure` binary from the Nginx buildpack at run-time before nginx starts